tests/acceptance/home/contribution/new/new-test.js

```diff
@@ -0,0 +1,87 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { click, fillIn } from '@ember/test-helpers';
+import { selectChoose } from 'ember-power-select/test-support';
+
+import { add, format } from 'date-fns';
+
+module('Acceptance | contribution/new', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    this.sessionService = this.owner.lookup('service:session');
+    const store = await this.owner.lookup('service:store');
+
+    const loggedUser = await this.server.create('user');
+    const loggedUserModel = await store.findRecord('user', loggedUser.id);
+
+    this.set('store', store);
+    this.set('sessionService.currentUser', loggedUserModel);
+
+    window.localStorage.setItem(
+      'storage:logged-as',
+      JSON.stringify({ id: loggedUserModel.id })
+    );
+  });
+
+  test('visiting /contribution/new', async function (assert) {
+    await visit(`/contribution/new`);
+    assert.strictEqual(currentURL(), `/contribution/new`);
+
+    assert.dom('[data-test-contribution-title]').exists();
+    assert.dom('[data-test-contribution-deadline]').exists();
+    assert.dom('[data-test-contribution-privacy]').exists();
+    assert.dom('.user-select').exists();
+    assert.dom('[data-test-add-contributor]').exists();
+    assert.dom('[data-test-amount]').exists();
+    assert.dom('[data-test-create]').exists();
+  });
+
+  test('adding new contribution', async function (assert) {
+    await this.server.create('user');
+    await this.server.create('user');
+    await this.server.create('user');
```

Comment:
🤔  + I'm not sure it is asynchronous method:
```suggestion
    this.server.createList('user', 3);
```